src/PrestaShopBundle/Controller/Admin/Sell/Order/CartController.php

```diff
@@ -281,13 +340,18 @@ public function deleteCartRuleAction(int $cartId, int $cartRuleId)
      */
     public function addProductAction(Request $request, int $cartId): JsonResponse
     {
-        $productId = $request->request->getInt('productId');
-        $quantity = $request->request->getInt('quantity');
-        $combinationId = $request->request->getInt('combinationId');
+        $productId = $request->request->getInt('product_id');
```

Comment:
How come in some functions you change snake case to camel case (`editAddressesAction`) but here you do the opposite? It would be more consistent to use the same naming convention everywhere (or at least try to ^^)

Reply:
yeah, might seem so, but this is the case when i submit a form and form names are snake_cased :smile:
So these interferes a bit. The reason is happening here because i got rid of some js code by using FormData instead of manually collecting form fields.

Reply:
Mmm, I understand the problem Although it's easier this way I don't think the parameter convention of our APIs should be defined by the implementation in javascript We should have a strictly defined convention
I guess here we have the same question about which convention we use in forms as well

Reply:
In this case i could name form inputs in camelCase as its manually built in twig, but in different case where its built with symfony form builder we wont be able to change it so easy. So idk, should i now leave it snake_cased or change to camelCase?

Reply:
Can be refactored after QA check 👍